_board/m5stack_cardputer.md

```diff
@@ -0,0 +1,44 @@
+---
+layout: download
+board_id: "m5stack_cardputer"
+title: "Dial - Smart Rotary Knob with Touch Screen Download"
+name: "Dial - Smart Rotary Knob with Touch Screen"
```

Comment:
Looks like these might be leftover accidentally from the Dial device posting.

Reply:
Thanks, should be fixed now. And your stream sparked my interest. :-)